### ISSUE TITLE
The requested module 'cheerio' does not provide an export named 'default'

### ISSUE THREAD

**Issue by r-erd** [tanishqmanuja/apkmirror-downloader#18]
Hi, 
I am having trouble using the library.  I just copied the example code into a file and tried to run it (`node example.js`)
I tested installing it from pnpm and npm, neither works.

I am getting the following error:

```
            ^
SyntaxError: The requested module 'cheerio' does not provide an export named 'default'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:171:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:254:5)
    at async ModuleLoader.import (node:internal/modules/esm/loader:474:24)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:119:5)

Node.js v22.1.0
```

Am I missing something or is there an issue with the library that has gone unnoticed?

**Comment by tanishqmanuja** [comment]
What sample code ?

**Comment by tanishqmanuja** [comment]
Thanks for the issue, should be resolved now  with commit (07f914b273da633c6357f3f39b3d951677c6fa61)
